source/ref_gl/r_vbo.c

```diff
@@ -21,13 +21,8 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #include "r_local.h"
 #include "qmesa.h"
 
-/*
-=========================================================
-
-VERTEX BUFFER OBJECTS
-
-=========================================================
-*/
+R_CreateMeshVBOFn R_CreateMeshVBO;
+R_ReleaseMeshVBOFn R_ReleaseMeshVBO;
 
 typedef struct vbohandle_s
```

Comment:
how much do you save avoiding heap allocation and using a pool like this? @ladyhavoc